examples/logging/advanced_requests.md

```diff
@@ -0,0 +1,65 @@
+By default django logs [4xx and 5xx requests](https://docs.djangoproject.com/en/3.2/topics/logging/#django-request) 
+but is missing details such as the request user name.  This can be helpful in debugging. This example 
+shows how the request user can be added to the logs.
```

Comment:
I think it'd be useful in each of these example files to include a sample log snippet showing what sorts of messages might be generated and how they're formatted.